client/my-sites/upgrades/checkout-thank-you/header.jsx

```diff
@@ -21,10 +21,7 @@ const CheckoutThankYouHeader = React.createClass( {
 	propTypes: {
 		isDataLoaded: React.PropTypes.bool.isRequired,
 		primaryPurchase: React.PropTypes.object,
-		selectedSite: React.PropTypes.oneOfType( [
-			React.PropTypes.object,
-			React.PropTypes.bool
-		] )
+		selectedSite: React.PropTypes.object.isRequired
```

Comment:
It looks like we should actually update the `selectedSite` prop type in all `CheckoutThankYou` components. Good catch!